forge-gui/res/cardsfolder/upcoming/saruman_of_many_colors.txt

```diff
@@ -0,0 +1,14 @@
+Name:Saruman of Many Colors
+ManaCost:3 W U B
+Types:Legendary Creature Avatar Wizard
+PT:5/4
+K:Ward:Discard<1/Enchantment;Instant;Sorcery>
+T:Mode$ SpellCast | ValidCard$ Card.YouCtrl | ValidActivatingPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigMill | ActivatorThisTurnCast$ EQ2 | TriggerDescription$ Whenever you cast your second spell each turn, each opponent mills two cards. When one or more cards are milled this way, exile target enchantment, instant, or sorcery card with equal or lesser mana value than that spell from an opponent's graveyard. Copy the exiled card. You may cast the copy without paying its mana cost.
+SVar:TrigMill:DB$ Mill | Defined$ Opponent | NumCards$ 2 | RememberMilled$ True | SubAbility$ DBExile
```

Comment:
Use release notes:
> You don't choose a target for Saruman of Many Colors's ability at the time it triggers. Rather, a second "reflexive" ability triggers when one or more cards are milled this way. You choose a target for that ability as it goes on the stack. Each player may respond to this triggered ability as normal.

Reply:
I think with the immediatetrigger the `SVar:X:TriggeredCard$CardManaCost` doesn't work anymore ?